modules/exploits/windows/local/lexmark_driver_privesc.rb

```diff
@@ -0,0 +1,187 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = NormalRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::EXE
+  include Msf::Post::Windows::Priv
+  include Msf::Exploit::FileDropper
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Lexmark Driver Privilege Escalation',
+        'Description' => %q{
+          Various Lexmark Universal Printer drivers as listed at advisory TE953
+          allow low-privileged authenicated users to elevate their privileges to
+          SYSTEM on affected Windows systems by modifying the XML file at
+          C:\ProgramData\Lexmark Universal v2\Universal Color Laser.gdl
```

Comment:
If we change this in the documentation then we should also change this in the module as well.
```suggestion
          C:\\ProgramData\\<driver name>\\Universal Color Laser.gdl
```